diff --git a/node_modules/react-native-awesome-pin/PinScreen.js b/node_modules/react-native-awesome-pin/PinScreen.js
index f065293..cbfbe27 100644
--- a/node_modules/react-native-awesome-pin/PinScreen.js
+++ b/node_modules/react-native-awesome-pin/PinScreen.js
@@ -314,6 +314,6 @@ const styles = StyleSheet.create({
     flex: null,
     color: "#FFF",
     fontSize: 17,
-    fontWeight: "bold"
+    // fontWeight: "bold"
   }
 });
